client/app.jsx

```diff
@@ -53,12 +53,12 @@ class App extends React.Component {
             // console.log('rateObj: ', this.state.ratingObj);
             for (var key in rateObj) {
               // console.log('numKey');
-              result = result + Number(key)*Number(rateObj[key]);
+              result = result + Number(key) * Number(rateObj[key]);
               numRating = numRating + Number(rateObj[key]);
             }
             // console.log('result: ', result);
             // console.log('numRating: ', numRating);
-            var currRating = result/numRating;
+            var currRating = result / numRating;
```

Comment:
@anteofabris the last functionality I have left to do is the star ratings so I'll ask you about that tomorrow